### PR TITLE
Untracked: Center titles and footer to plot area

### PR DESCRIPTION
**Pull request by JustinCCYap**

### CODE REVIEW

File: theSrc/scripts/LabeledScatter.js [hunk 1]
```diff
@@ -582,7 +582,7 @@ class LabeledScatter {
         .select('.cursor-pointer')
         .attr('x', 0)
         .attr('y', 0)
-        .attr('transform', `translate(${0.5 * this.width},${this.height - footer_height - footer_padding})`)
+        .attr('transform', `translate(${this.titleX('Center of plot area')},${this.height - footer_height - footer_padding})`)
```
Comment (chschan) [inline theSrc/scripts/LabeledScatter.js:585]: Why is this always using `this.titleX('Center of plot area')`? What if I change the alignment of the footer?

File: R/rhtmlCombinedScatter.R [hunk 2]
```diff
@@ -325,12 +325,12 @@ CombinedScatter <- function(
...
     subtitle.font.family = "Arial",
     subtitle.font.size = 12,
-    subtitle.alignment = "Center",
+    subtitle.alignment = "Center of plot area",
     title = "",
     title.font.color = rgb(44, 44, 44, maxColorValue = 255),
     title.font.family = "Arial",
     title.font.size = 16,
-    title.alignment = "Center",
+    title.alignment = "Center of plot area",
```
Comment (chschan) [inline R/rhtmlCombinedScatter.R:333]: Can you update the R docs to list the possible values for for `title.alignment`, `subtitle.alignment`, and `footer.alignment`.

File: theSrc/scripts/LabeledScatter.js (hunk 1 above)
Comment (JustinCCYap) [reply theSrc/scripts/LabeledScatter.js:585]: The footer doesn't have an alignment setting. I could include it but it won't be exposed to users at the moment. We could include it in the future but I thought eventually footers will be drawn by Displayr.

File: R/rhtmlCombinedScatter.R (hunk 2 above)
Comment (JustinCCYap) [reply R/rhtmlCombinedScatter.R:333]: Done in https://github.com/Displayr/rhtmlCombinedScatter/pull/50/commits/b4dee8635064f5195df7d99265672441d5f46ece

File: theSrc/scripts/LabeledScatter.js (hunk 1 above)
Comment (chschan) [reply theSrc/scripts/LabeledScatter.js:585]: Ah right. I forgot that the footers will eventually be drawn by Displayr

**Review by chschan — APPROVED** [review]

**Pull request merged**